packages/scripts/README.md

```diff
@@ -54,6 +54,12 @@ This is how you execute the script with presented setup:
 
 * `npm run build` - builds the code for production.
 
+_Options_:
+
+- `--WP_BUNDLE_ANALYZER` : This flag is used to enables utility that represents bundle content as convenient interactive zoomable treemap.
```

Comment:
As I understand it, these are interpreted by the `wp scripts` script as [environment variables](https://en.wikipedia.org/wiki/Environment_variable), not as options flags.

In other words, instead of:

```
wp scripts build --WP_BUNDLE_ANALYZER
```

I think the correct usage is...

```
WP_BUNDLE_ANALYZER=true wp scripts
```

Please correct me if I'm mistaken.

Reply:
Hi! Thanks for comments!
I am not sure which way is correct to use `--WP_BUNDLE_ANALYZER`. But for the `WP_LIVE_RELOAD_PORT`, I have tried `npm run build --WP_LIVE_RELOAD_PORT=12345` and `npm run build WP_LIVE_RELOAD_PORT=12345`. The first one works but the second one fails and the error message is

> Insufficient number of arguments or no entry found. Alternatively, run 'webpack(-cli) --help' for usage info.

According to the documentation of [Environment Variables](https://webpack.js.org/guides/environment-variables/) and [Environment Options](https://webpack.js.org/api/cli/#environment-options) in web pack, I think we can use the double hyphen to specify the variables including `WP_BUNDLE_ANALYZER`, `WP_BUNDLE_ANALYZER` and `WP_DEVTOOL`.

Reply:
@aduth is correct, they should be rather provided before the command. Unless we replicate what @draganescu did for e2e tests and introduce new CLI args, convert them to env variables in node and filter them out before passing to webpack.